marklogic-data-hub-central/ui/src/App.scss

```diff
@@ -145,3 +145,11 @@ div#error-list div.ant-collapse div.ant-collapse-item div.ant-collapse-content d
   background: #44499C;
   color: #fff;
 }
+
+.has-error .ant-input, .has-error .ant-input:focus{
+  border-color: #B32424;
```

Comment:
This alert color should be applied globally at the theme level.

Reply:
Fixed. Thanks.